ivy/container/activations.py

```diff
@@ -1,9 +1,38 @@
+# global
+from typing import Optional, Union
+
 # local
+import ivy
 from ivy.container.base import ContainerBase
 
 # ToDo: implement all methods here as public instance methods
 
 
 # noinspection PyMissingConstructor
 class ContainerWithActivations(ContainerBase):
-    pass
+    def softplus(
```

Comment:
Hi! Great job! But before merging, could you please add the container arguments needed? You can see an example [here](https://github.com/unifyai/ivy/blob/8d1eef71522be7f98b601e5f97bb2c54142795b3/ivy/container/elementwise.py#L1240)

Reply:
Hey Zoe, thanks for the feedback! I added the container args-- let me know if anything else needs changing.